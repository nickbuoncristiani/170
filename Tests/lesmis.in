77
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77
9 53 19 25 60 58 44 23 73 47 18 36 11 39 20 57 32 34 40 52 71 8 75 48 31
13
x 1.16439 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.16439 x 1.99975 1.39076 1.9493 1.16412 1.94179 1.76881 1.8194 1.73307 1.74249 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.99975 x 1.52231 x x x x x x 1.80666 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.39076 1.52231 x x x x x x x 1.41739 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.9493 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.16412 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.94179 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.76881 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.8194 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.73307 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.74249 1.80666 1.41739 x x x x x x x 1.85469 1.22221 1.75587 1.30284 1.11275 1.32625 1.89913 1.43071 1.96434 1.53988 1.48057 1.53833 1.79522 1.52309 1.37473 1.80778 1.92116 1.13813 1.10813 1.02654 1.99803 1.9044 1.86495 1.35539 1.28304 1.07681 1.08743 1.39804 1.05014 1.46162 1.61633 1.1653 1.79194 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.85469 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.22221 x x x x x 1.04752 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.75587 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.30284 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.11275 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.32625 x 1.04752 x x x x 1.49043 1.88142 x 1.9418 x 1.65498 1.91762 x x x x x x x x x x x x x x x x x x x x 1.70677 1.98096 1.11335 1.77181 1.06921 1.00067 1.61802 1.21022 x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.89913 x x x x x 1.49043 x 1.71111 1.78554 1.36263 x x x x x x x x x x x x x x x x x x 1.22981 1.70931 1.12837 x x x x x x x x x x 1.7171 1.35872 1.68789 x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.43071 x x x x x 1.88142 1.71111 x 1.72148 1.7765 x x x x x x x x x x x x 1.7431 x x 1.96133 x x 1.01851 1.98023 1.20875 1.14539 x x x x x x x x x 1.94071 1.35913 x 1.96336 1.98185 1.33967 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.96434 x x x x x x 1.78554 1.72148 x 1.05444 x x x x x x x x x x 1.18451 x x 1.51689 1.25592 1.30664 x x x x x x 1.7007 x 1.51422 x x x x x x x x x x x x 1.73864 x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.53988 x x x x x 1.9418 1.36263 1.7765 1.05444 x 1.83388 1.10904 1.79352 x 1.44788 x x x x x 1.78556 x 1.62273 x x x 1.72939 x 1.70368 1.25181 1.19026 1.64818 1.34296 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.48057 x x x x x x x x x 1.83388 x x x x x x x x x x x 1.57358 x x x x x x x x x x x x x x x x x x x x x x x x x x 1.78616 x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.53833 x x x x x 1.65498 x x x 1.10904 x x x x x 1.28183 1.24802 1.87411 1.74428 1.55031 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.79522 x x x x x 1.91762 x x x 1.79352 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.24729 x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.52309 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.37473 x x x x x x x x x 1.44788 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.80778 x x x x x x x x x x x 1.28183 x x x x 1.56024 1.55605 1.59344 1.51658 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.92116 x x x x x x x x x x x 1.24802 x x x 1.56024 x 1.80919 1.22607 1.41655 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.13813 x x x x x x x x x x x 1.87411 x x x 1.55605 1.80919 x 1.77775 1.06173 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.10813 x x x x x x x x x x x 1.74428 x x x 1.59344 1.22607 1.77775 x 1.11216 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.02654 x x x x x x x x x x x 1.55031 x x x 1.51658 1.41655 1.06173 1.11216 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.99803 x x x x x x x x 1.18451 1.78556 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.9044 x x x x x x x x x x 1.57358 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.86495 x x x x x x x 1.7431 x 1.62273 x x x x x x x x x x x x x x x 1.23611 1.01544 1.51191 1.85633 1.58275 x 1.6713 x x x x x x x x x x x x x x 1.56631 x x x 1.69932 1.86004 1.66528 x 1.9308 1.27693 1.2792 1.50332 1.4535 1.45067 1.50495 1.20562 1.65587 x x x
x x x x x x x x x x 1.35539 x x x x x x x x 1.51689 x x x x x x x x x x x x x x x 1.44435 1.83821 x x x x x x x x x x x x x x x x x 1.91569 x x x 1.87875 x x x x x x x x x x x x x x x 1.83876 x x
x x x x x x x x x x 1.28304 x x x x x x x x 1.25592 x x x x x x x x x x x x x x 1.44435 x 1.57188 x x x x x x x x x x x x x x x x x x x x x 1.34094 x 1.58692 x x x x x x x x x x x x x x 1.88579 x
x x x x x x x x x x 1.07681 x x x x x x x 1.96133 1.30664 x x x x x x x x x x x x x 1.23611 1.83821 1.57188 x 1.64447 1.40542 x x x x x x 1.07848 x x x x x x 1.06603 x x 1.49751 x x 1.88549 x x 1.89648 1.12656 x x 1.60137 x 1.6909 1.29178 1.5325 x x x x 1.95029 x x
x x x x x x x x x x 1.08743 x x x x x x x x x 1.72939 x x x x x x x x x x x x 1.01544 x x 1.64447 x 1.63006 x x 1.51135 x x x x x x x x x x x x x x x x x x x 1.63859 1.48775 x x 1.28002 1.59126 1.83983 1.18482 1.90665 1.44905 x x 1.22687 x x x
x x x x x x x x x x 1.39804 x x x x x x x x x x x x x x x x x x x x x x 1.51191 x x 1.40542 1.63006 x x x x x x x x x x x x x x x x x x x x x x x 1.59144 1.91938 x x 1.19613 1.10568 1.14045 1.88277 1.43943 1.29924 x x 1.39264 x x x
x x x x x x x x x x 1.05014 x x x x x x 1.22981 1.01851 x 1.70368 x x x x x x x x x x x x 1.85633 x x x x x x 1.52739 1.04141 1.51278 x x x x x x x x x 1.3748 x x x x 1.67239 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.46162 x x x x x x 1.70931 1.98023 x 1.25181 x x x x x x x x x x x x 1.58275 x x x x x 1.52739 x 1.52498 1.60749 x x x x x x x x x 1.51911 x x x x 1.96001 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.61633 x x x x x x 1.12837 1.20875 x 1.19026 x x x x x x x x x x x x x x x x 1.51135 x 1.04141 1.52498 x 1.08902 x x x x x x x x x 1.2241 x x x x 1.61414 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.1653 x x x x x x x 1.14539 x 1.64818 x x x x x x x x x x x x 1.6713 x x x x x 1.51278 1.60749 1.08902 x x x x x x x x x x 1.83913 x x x x 1.85369 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.79194 x x x x x x x x 1.7007 1.34296 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.70677 x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.85321 1.64606 1.72853 1.73782 1.23264 1.59369 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.98096 x x 1.51422 x x x x x x x x x x x x x x x x 1.07848 x x x x x x x 1.85321 x 1.10657 1.34021 1.71135 1.29695 1.54489 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.11335 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.64606 1.10657 x 1.15962 1.76463 1.44905 1.31399 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.77181 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.72853 1.34021 1.15962 x 1.51034 1.98461 1.76775 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.06921 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.73782 1.71135 1.76463 1.51034 x 1.73229 1.11416 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.00067 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.23264 1.29695 1.44905 1.98461 1.73229 x 1.37914 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.61802 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.59369 1.54489 1.31399 1.76775 1.11416 1.37914 x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.21022 x x x x x x 1.24729 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.7171 1.94071 x x x x x x x x x x x x x x x x x 1.06603 x x 1.3748 1.51911 1.2241 1.83913 x x x x x x x x x x 1.741 x x x 1.34415 x x x 1.28732 1.12041 x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.35872 1.35913 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.741 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.68789 x x x x x x x x x x x x x x x x 1.91569 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.96336 x x x x x x x x x x x x x x x x x 1.49751 x x x x x x x x x x x x x x x x x x x x x x x 1.8428 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.98185 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.33967 x x x x x x x x x x x x x x 1.56631 x x x x x 1.67239 1.96001 1.61414 1.85369 x x x x x x x x x 1.34415 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x 1.73864 x x x x x x x x x x x x x x 1.87875 1.34094 1.88549 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.78616 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.58692 x x x x x x x x x x x x x x x x x x x 1.8428 x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.69932 x x 1.89648 1.63859 1.59144 x x x x x x x x x x x x x 1.28732 x x x x x x x x x 1.54485 x x 1.54352 x 1.60485 1.11549 1.98343 x x x x x x 1.746
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.86004 x x 1.12656 1.48775 1.91938 x x x x x x x x x x x x x 1.12041 x x x x x x x x 1.54485 x x x 1.72864 1.71889 1.88675 1.9912 1.48362 1.83906 x x 1.76178 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.66528 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.50671 x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.50671 x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.9308 x x 1.60137 1.28002 1.19613 x x x x x x x x x x x x x x x x x x x x x x 1.54352 1.72864 x x x 1.84677 1.86254 1.05189 1.90519 1.66324 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.27693 x x x 1.59126 1.10568 x x x x x x x x x x x x x x x x x x x x x x x 1.71889 x x 1.84677 x 1.29838 1.50917 1.91211 1.57808 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.2792 x x 1.6909 1.83983 1.14045 x x x x x x x x x x x x x x x x x x x x x x 1.60485 1.88675 x x 1.86254 1.29838 x 1.74193 1.19227 1.9778 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.50332 x x 1.29178 1.18482 1.88277 x x x x x x x x x x x x x x x x x x x x x x 1.11549 1.9912 x x 1.05189 1.50917 1.74193 x 1.26464 1.34672 x x 1.64729 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.4535 x x 1.5325 1.90665 1.43943 x x x x x x x x x x x x x x x x x x x x x x 1.98343 1.48362 x x 1.90519 1.91211 1.19227 1.26464 x 1.59171 x x 1.74262 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.45067 x x x 1.44905 1.29924 x x x x x x x x x x x x x x x x x x x x x x x 1.83906 x x 1.66324 1.57808 1.9778 1.34672 1.59171 x x x 1.15422 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.50495 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.4602 x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.20562 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.4602 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.65587 x x x 1.22687 1.39264 x x x x x x x x x x x x x x x x x x x x x x x 1.76178 x x x x x 1.64729 1.74262 1.15422 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.83876 x 1.95029 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.88579 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.746 x x x x x x x x x x x x x x x
