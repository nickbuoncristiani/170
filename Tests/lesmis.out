13 11 34 11 2 11 17 47 48 17 11 13
6
11 36 39 40 11 44 75 19 23 25 58 57 60 31 32
34 73 34 71
2 9 8
17 52
47 47
48 48
